build_tools/kokoro/gcp_ubuntu/cmake/linux/x86-swiftshader/common.cfg

```diff
@@ -0,0 +1,19 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Common configuration for Kokoro builds that run cmake on linux.
```

Comment:
Comment doesn't seem accurate. Honestly maybe we should just delete these comments. I don't think they add anything and we keep copypastaing them.

Reply:
Yeah. The info is essentially represented via the file path. Each time we copy we need to tweak it, it's a bit annoying. Deleting it.